### PR TITLE
[RHCLOUD-24387] Replace SVG by PNG images on emails

### PR DESCRIPTION
**Pull request by g-duval**
Because at least Gmail can't render SVG.

As workaround, and for stage validation, we added images directly on emails bodies, then they will have to be hosted on https://console.redhat.com/apps/frontend-assets/...

### CODE REVIEW

File: engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html [hunk 1]
```diff
@@ -82,14 +85,13 @@
...
-                        <path><title>Incident</title></path>
-                    </svg>
+                    {#if isIncident && (true == isIncident || "true" == isIncident)}
+                        <div class="fa-exclamation-circle" title="Incident"></div>
+                    {#else}
+                        <div class="empty-div-16"></div>
+                    {/if}
                 {/let}
               <a target="_blank" href="{environment.url}/insights/advisor/recommendations/{rec}">{action.context.advisor.resolved_recommendations.get(rec).rule_description}</a>
-              {#if action.context.advisor.resolved_recommendations.get(rec).has_incident && true == action.context.advisor.resolved_recommendations.get(rec).has_incident}
```
Comment (g-duval) [inline engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html:90]: This "if" had no effect on previous version, I cleaned it.

File: engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html [hunk 2]
```diff
@@ -121,14 +123,13 @@
...
-                        <path><title>Incident</title></path>
-                    </svg>
+                    {#if isIncident && (true == isIncident || "true" == isIncident)}
+                        <div class="fa-exclamation-circle" title="Incident"></div>
+                    {#else}
+                        <div class="empty-div-16"></div>
+                    {/if}
                 {/let}
                 <a target="_blank" href="{environment.url}/insights/advisor/recommendations/{rec}">{action.context.advisor.deactivated_recommendations.get(rec).rule_description}</a>
-                {#if action.context.advisor.deactivated_recommendations.get(rec).has_incident && true == action.context.advisor.deactivated_recommendations.get(rec).has_incident}
```
Comment (g-duval) [inline engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html:129]: This "if" had no effect on previous version, I cleaned it.

File: engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html [hunk 3]
```diff
@@ -82,14 +85,13 @@
...
+                    {#if isIncident && (true == isIncident || "true" == isIncident)}
+                        <div class="fa-exclamation-circle" title="Incident"></div>
+                    {#else}
+                        <div class="empty-div-16"></div>
+                    {/if}
                 {/let}
               <a target="_blank" href="{environment.url}/insights/advisor/recommendations/{rec}">{action.context.advisor.resolved_recommendations.get(rec).rule_description}</a>
-              {#if action.context.advisor.resolved_recommendations.get(rec).has_incident && true == action.context.advisor.resolved_recommendations.get(rec).has_incident}
-                <span class="rh-incident">Incident</span>
-              {/if}
```
Comment (josejulio) [inline engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html:92]: Is this intended? Remove the "Incident" text from the email?

File: engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html [hunk 4]
```diff
@@ -121,14 +123,13 @@
...
+                    {#if isIncident && (true == isIncident || "true" == isIncident)}
+                        <div class="fa-exclamation-circle" title="Incident"></div>
+                    {#else}
+                        <div class="empty-div-16"></div>
+                    {/if}
                 {/let}
                 <a target="_blank" href="{environment.url}/insights/advisor/recommendations/{rec}">{action.context.advisor.deactivated_recommendations.get(rec).rule_description}</a>
-                {#if action.context.advisor.deactivated_recommendations.get(rec).has_incident && true == action.context.advisor.deactivated_recommendations.get(rec).has_incident}
-                    <span class="rh-incident">Incident</span>
-                {/if}
```
Comment (josejulio) [inline engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html:131]: Same as above

File: engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html (hunk 3 above)
Comment (josejulio) [reply engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html:92]: Just saw your comment :grin:

File: engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html (hunk 4 above)
Comment (josejulio) [reply engine/src/main/resources/templates/Advisor/dailyEmailBodyV2.html:131]: Same

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**